xapian-core/tests/termgentest.cc

```diff
@@ -171,8 +171,13 @@ static const test test_simple[] = {
     { "stop_stemmed",
     	"The stop words.", "stop[2] the[1] words[3]" },
```

Comment:
Should we be running a test with stemming and `stop_stemmed`? Because this has stemming disabled, right? (Because of the `none` at line 165.) Ideally we want to test the different combinations of stemming and stopping options.